### DISCUSSION TITLE
Difference between package "code" in pacman and "vscodium" in AUR?

### DISCUSSION THREAD

**Discussion by ghost** [VSCodium/vscodium#869] [Q&A] [answered]
I've been messing with Arch in a VM and I noticed that the package "code" in pacman is Microsoft's "Code - OSS" instead of branded VSCode. Is there any difference between this package and VSCodium in the AUR?

**Comment by elgatofeliz** [comment]
live server auto-reload doesnt work in Code-Oss, but does in VSCodium

**Comment by paulcarroty** [comment, marked as answer]
The Arch Code is a clean VSC build without the VSCodium patches like disabled telemetry or custom marketplace.
